reserved_keyword_checker/tests/test_files/invalid_overrides.yml

```diff
@@ -0,0 +1,5 @@
+---
+STITCH:
+    - BasicModel.first_field
+    - BasicModel. second_field
```

Comment:
Is this space intentional?

Reply:
Yes, it's intentionally improperly formatted for a test.